python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -362,7 +345,10 @@ def __ge__(self, obj_list):
         return self._list.__ge__(self.__getListToCompare(obj_list))
 
     def __getitem__(self, index):
-        return self._list.__getitem__(index)
+        returnable = self._list.__getitem__(index)
+        if isinstance(returnable, Node):
+            returnable._setParent(self._getParent())
+        return returnable
```

Comment:
I can remove this, typo. was left in when tracking down some madness due to missing parents